sipa/static/css/agdsn-bootstrap-social.css

```diff
@@ -0,0 +1,67 @@
+.btn-social-icon {
+    position: relative;
+    padding-left: 44px;
+    text-align: left;
+    write-space: nowrap;
```

Comment:
typo, you meant `white-space`. However, you can most likely omit this, as the text the element contains is a single char.